tests/test_httpie.py

```diff
@@ -218,3 +219,21 @@ def test_json_input_preserve_order(httpbin_both):
     assert HTTP_OK in r
     assert r.json['data'] == \
         '{"order": {"map": {"1": "first", "2": "second"}}}'
+
+
+@responses.activate
+def test_GET_encoding_detection_from_content():
+    url = 'http://example.org'  # Note: URL never fetched
+    body = '<?xml version="1.0" encoding="UTF-8"?><category>Financiën</category>'.encode('utf8')
+    responses.add(responses.GET, url, body=body, content_type='text/xml')
```

Comment:
I am not sure if `httpbin` does allow doing such things. @jakubroztocil Let me know if you think of a way of doing that without a new dependency (I am OK with that, but if we can keep requirements light, it is always good).
And in the other way I really like `responses`, it is very easy to mock calls without touching `requests` internals ourselves.

Reply:
I'll see if I can get rid of `responses`, I'm not comfortable to add a `requests`-specific module to mock calls. Trying to stick with `httpbin` would be better.

Reply:
Finally, it is not worth losing time on it. I did not find how to do it and at the end `httpbin` is also tight to `resquests`.